### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by vedant-z**
-   Please make sure the below checklist is followed for Pull Requests.

Fixes #

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by nicolas63** [comment]
Your code is right but you need to declare the const from this file https://github.com/jhipster/generator-jhipster/blob/main/lib/constants/priorities.cjs

Like this line https://github.com/jhipster/jhipster-dotnetcore/blob/main/generators/client/files-common.js#L22

**Comment by vedant-z** [comment]
@nicolas63 Thank you for supporting and guiding me 
I know you might be frustrated by the checks failing as am I.

File: generators/app/index.js [hunk 1]
```diff
@@ -112,35 +120,35 @@ module.exports = class extends AppGenerator {
...
         return super._preparing();
     }
 
-    get default() {
+    get [DEFAULT_PRIORITY]() {
         return super._default();
     }   
 
-    get writing() {
+    get [WRITIING_PRIORITY]() {
```
Comment (nicolas63) [inline generators/app/index.js:143]: Typo Wriiting => Writing

File: generators/app/index.js [hunk 2]
```diff
@@ -1,15 +1,23 @@
 /* eslint-disable consistent-return */
 const chalk = require('chalk');
+
 const AppGenerator = require('generator-jhipster/generators/app');
+
+const constants = require('generator-jhipster/lib/constants/priorities.cjs');
```
Comment (nicolas63) [inline generators/app/index.js:6]: correct the indentation (eslint can help you) install it with 
```
npm install eslint --global 
```
after that you can run eslint --fix .

**Review by nicolas63 — CHANGES_REQUESTED** [review]

**Comment by nicolas63** [comment]
> @nicolas63 Thank you for supporting and guiding me I know you might be frustrated by the checks failing as am I.

no worry about this, i'm here to help newcomer

File: generators/app/index.js [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
+
 const packagejs = require('../../package.json');
+
 const prompts = require('./prompts');
 
+
+const { INITIALIZING_PRIORITY, PROMPTING_PRIORITY, CONFIGURING_PRIORITY, COMPOSING_PRIORITY, LOADING_PRIORITY, PREPARING_PRIORITY, DEFAULT_PRIORITY, WRITING_PRIORITY, POST_WRITING_PRIORITY, END_PRIORITY } = constants;
+
 module.exports = class extends AppGenerator {
     constructor(args, opts) {
```
Comment (nicolas63) [inline generators/app/index.js:16]: register priority prefix in the constructor 

```
constructor(args, opts, features) {
        super(args, opts, { fromBlueprint: true, taskPrefix: PRIORITY_PREFIX, ...features }); // fromBlueprint variable is important
    }
```

File: generators/app/index.js [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
 const AppGenerator = require('generator-jhipster/generators/app');
+
+const constants = require('generator-jhipster/lib/constants/priorities.cjs');
+
 const packagejs = require('../../package.json');
+
 const prompts = require('./prompts');
 
+
+const { INITIALIZING_PRIORITY, PROMPTING_PRIORITY, CONFIGURING_PRIORITY, COMPOSING_PRIORITY, LOADING_PRIORITY, PREPARING_PRIORITY, DEFAULT_PRIORITY, WRITING_PRIORITY, POST_WRITING_PRIORITY, END_PRIORITY } = constants;
```
Comment (nicolas63) [inline generators/app/index.js:13]: add priority prefix here, PRIORITY_PREFIX

**Review by nicolas63 — CHANGES_REQUESTED** [review]

**Comment by nicolas63** [comment]
@vedant-z do you need help ?

**Comment by vedant-z** [comment]
I think I can't solve this issue 😥

I have done everything as said by you but still it's not passing thr checks 

What should I do now @nicolas63

**Review by nicolas63 — CHANGES_REQUESTED** [review]

**Comment by nicolas63** [comment]
> I think I can't solve this issue 😥
> 
> I have done everything as said by you but still it's not passing thr checks
> 
> What should I do now @nicolas63

 see the last review, it's a small problem with the constructor

**Comment by nicolas63** [comment]
Looks good to me, just rebase your branch and i merge your PR

**Comment by nicolas63** [comment]
@vedant-z any update ?

**Comment by vedant-z** [comment]
Ya I will rebase and make PR

**Pull request closed without merging**